### DISCUSSION TITLE
Height of ruled lines

### DISCUSSION THREAD

**Discussion by dual3zw** [xournalpp/xournalpp#3375] [Q&A] [answered]
Is there any way to change the height of ruled lines on new pages or page model?

**Comment by rolandlo** [comment, marked as answer]
Unfortunately changing the height of ruled lines is not possible currently. It should be defenitely be implemented in my opinion.

**Reply by dual3zw** [reply]
Ohh, unfortunately. But that doesn't take the shine off this app! It's wonderful, highly customizable and lightweight! Congratulations to the developers!
